src/dicomweb_client/ext/gcp/session_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Session management utilities for Google Cloud Platform (GCP)."""
+from typing import Optional, Any
+
+import google.auth
+from google.auth.transport import requests as google_requests
```

Comment:
I wondering whether we should try and catch import errors to inform users about the extra vendor-specific dependency. For example:

```python
try:
    import google.auth
    from google.auth.transport import requests as google_requests
except ImportError:
    raise ImportError(
        'To use the gcp extension install the package as follows: '
        '"pip install dicomweb-client[gcp]"'
    )
```

What do you think?

Reply:
That makes sense, thank you! I've moved the check here from `session_utils.create_session_from_gcp_credentials()`. Tweaked the phrasing a bit to indicate that the `gcp` extras are required specifically for the module `ext.gcp.session_utils` (and not necessarily the entire `ext.gcp` sub-package).